webpack/assets/javascripts/react_app/components/common/forms/TextField/__snapshots__/TextField.test.js.snap

```diff
@@ -1,34 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`TextField should default to a text field 1`] = `
-<Field
-  className=""
-  component={[Function]}
-  label="Name"
+<FormikConnect(FieldInner)
```

Comment:
Oh snap...  this snap looks broken

Reply:
This snap looks strange, but I think it is actually correct, the component returns a Formik field...

Reply:
but it returns the same formik component on different given props, it looks like a redundant snap to me

Reply:
?